test/sqs_lambda_to_batch_fargate/lambda/export_logs.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+import os
+import time
+
+import boto3
+
+s3_client = boto3.client("s3")
+s3_resource = boto3.resource("s3")
+logs_client = boto3.client("logs")
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def handler(event, context):
+    logger.info(f"INCOMING EVENT: {event}")
```

Comment:
Logging the raw event data should happen at `trace` or `debug` level. For the info level, I would maybe log... `detail-type` and `detail.status` as those are the main messages used?